Tests/AsyncHTTPClientTests/HTTPClientInternalTests+XCTest.swift

```diff
@@ -34,17 +34,11 @@ extension HTTPClientInternalTests {
             ("testRequestFinishesAfterRedirectIfServerRespondsBeforeClientFinishes", testRequestFinishesAfterRedirectIfServerRespondsBeforeClientFinishes),
             ("testProxyStreaming", testProxyStreaming),
             ("testProxyStreamingFailure", testProxyStreamingFailure),
-            ("testUploadStreamingBackpressure", testUploadStreamingBackpressure),
```

Comment:
do we have replacements for these tests?

Reply:
Yes. This is now tested way lower in the stack on the connection directly.